From 7f90e2cceda0458cf56026eb6ccffb961a47804b Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Date: Fri, 13 Oct 2017 15:49:32 +0200
Subject: [PATCH] IWYU: Include math.h for round(3).

math.h was being implicitly included, which can break the build with
alternative libc implementations.

Bug: None
Change-Id: I969b320b65d0f44abb33d3e1036cfbcb859a4952
Reviewed-on: https://webrtc-review.googlesource.com/9384
Reviewed-by: Tommi <tommi@webrtc.org>
Commit-Queue: Raphael Kubo da Costa (rakuco) <raphael.kubo.da.costa@intel.com>
Cr-Commit-Position: refs/heads/master@{#20292}
---
 p2p/base/port.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/p2p/base/port.cc b/p2p/base/port.cc
index a1b478d11..81aa0aadb 100644
--- a/third_party/webrtc/p2p/base/port.cc
+++ b/third_party/webrtc/p2p/base/port.cc
@@ -10,6 +10,8 @@
 
 #include "p2p/base/port.h"
 
+#include <math.h>
+
 #include <algorithm>
 #include <vector>
 
-- 
2.15.0.rc2

